civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -10,33 +10,88 @@ use Drupal\civihr_employee_portal\Helpers\HelperClass;
  */
 function civihr_employee_portal_css_alter(&$css) {
 
-    // Remove defaults system messages css file.
-    unset($css[drupal_get_path('module', 'system') . '/system.messages.css']);
+  // Remove defaults system messages css file.
+  unset($css[drupal_get_path('module', 'system') . '/system.messages.css']);
+
+  // Removing unwanted css files from civicrm pages.
+  if (civicrm_invoke()) {
```

Comment:
Why would you use `civicrm_invoke` to check if you are in /civicrm? Its [implementation](https://github.com/civicrm/civicrm-drupal/blob/7.x-master/civicrm.module#L367) seems considerable enough that I don't think it's a good idea to call it multiple times